src/assisted_test_infra/test_infra/helper_classes/kube_helpers/agent.py

```diff
@@ -187,6 +209,7 @@ def wait_till_all_agents_are_in_status(
         agents: List["Agent"],
         statusType: str,
```

Comment:
Argument name should be lowercase

Reply:
A major change in the existing code (all relevant and existing functions use statusType). Out of scope for this PR